### PR TITLE
[home] home accommodates selected artist UTM param

### PR DESCRIPTION
**Pull request by sarahscott**
Fixes https://github.com/artsy/collector-experience/issues/723

This makes sure that when a user taps an artist in a Works For You email, that artists' works are presented at the top of the Artists section when eigen is launched.

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Home/Components/ForYou/index.tsx [hunk 1]
```diff
@@ -12,17 +12,13 @@ interface DataSourceRow {
...
-}
-
 interface State {
   modules: any[]
   isRefreshing: boolean
   dataSource: ListViewDataSource
 }
 
-export class ForYou extends React.Component<Props, State> {
+export class ForYou extends React.Component<any, State> {
```
Comment (alloy) [inline src/lib/Scenes/Home/Components/ForYou/index.tsx]: Only the scroll hack prop needs to be removed, this component still needs those `ViewProperties` and `RelayProps`.

File: src/lib/AppRegistry.tsx [hunk 2]
```diff
@@ -67,6 +63,10 @@ const Gene: React.SFC<GeneProps> = ({ geneID, refineSettings: { medium, price_ra
   return <GeneRenderer {...initialProps} render={renderWithLoadProgress(Containers.Gene, initialProps)} />
 }
 
+const Home: React.SFC<{ selectedArtist: string }> = ({ selectedArtist }) => {
+  return <HomeScene selectedArtist={selectedArtist} />
+}
```
Comment (alloy) [inline src/lib/AppRegistry.tsx]: I’m not sure I understand why this is needed, it looks like both components basically expose one and the same prop. Can you elaborate?

File: src/lib/Scenes/Home/Components/ForYou/index.tsx [hunk 3]
```diff
@@ -12,13 +12,15 @@ interface DataSourceRow {
   data: any
 }
 
+interface Props extends ViewProperties, RelayProps {}
```
Comment (alloy) [inline src/lib/Scenes/Home/Components/ForYou/index.tsx]: This is fine, so just as an FYI; you can also define this as a ‘type’, removing the need to add empty curly braces:

```ts
type Props = ViewProperties & RelayProps
```

Comment (alloy) [inline src/lib/Scenes/Home/Components/ForYou/index.tsx]: This is fine, so just as an FYI; you can also define this as a ‘type’, removing the need to add empty curly braces:

```ts
type Props = ViewProperties & RelayProps
```

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/AppRegistry.tsx (hunk 2 above)
Comment (sarahscott) [reply src/lib/AppRegistry.tsx]: The prop isn't passed down to `Home` without being this explicit.

Comment (sarahscott) [reply src/lib/AppRegistry.tsx]: Actually I think I had something else missing before that was interfering here; this can be removed 😁

Comment (alloy) [reply src/lib/AppRegistry.tsx]: Ah ok, I was still scratching my head about what could have been different :D

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**